build.gradle.kts

```diff
@@ -53,6 +55,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-swing:$coroutinesVersion")
```

Comment:
I am facing an error while running plugin and generating tests for function. You don't need to fix it. But please check that you can run plugin from scratch with a new git clone

```
19:14:42: Executing task 'runIde'...

Cleaning up unclosed ZipFile for archive /Users/vadim-volodin/.gradle/caches/modules-2/files-2.1/com.intellij.remoterobot/robot-server-plugin/0.11.7/f634692a31a516ce76b0fa2dee749e82532fc3a4/robot-server-plugin-0.11.7.zip
> Task :extractIncludeProto
> Task :extractProto
> Task :generateProto

> Task :compileKotlin
w: /Users/vadim-volodin/IdeaProjects/UTBotCpp-CLion-plugin/src/main/kotlin/com/github/vol0n/utbotcppclion/client/GrpcClient.kt: (78, 26): This declaration is experimental and its usage should be marked with '@kotlinx.serialization.ExperimentalSerializationApi' or '@OptIn(kotlinx.serialization.ExperimentalSerializationApi::class)'
w: /Users/vadim-volodin/IdeaProjects/UTBotCpp-CLion-plugin/src/main/kotlin/com/github/vol0n/utbotcppclion/services/ProjectConfigurable.kt: (72, 80): Unchecked cast: JComponent! to JList<String>

> Task :compileJava
> Task :patchPluginXml
> Task :processResources
> Task :classes
> Task :instrumentCode
> Task :postInstrumentCode
> Task :inspectClassesForKotlinIC
> Task :jar
> Task :prepareSandbox
> Task :runIde
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.intellij.ide.ui.laf.darcula.DarculaLaf to field javax.swing.text.html.HTMLEditorKit.DEFAULT_STYLES_KEY
WARNING: Please consider reporting this to the maintainers of com.intellij.ide.ui.laf.darcula.DarculaLaf
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
2021-10-30 19:17:42,685 [   7597]   WARN - i.mac.MacOSApplicationProvider - No URL bundle (CFBundleURLTypes) is defined in the main bundle.
To be able to open external links, specify protocols in the app layout section of the build file.
Example: args.urlSchemes = ["your-protocol"] will handle following links: your-protocol://open?file=file&line=line 
2021-10-30 19:18:03,973 [  28885]   WARN - j.internal.DebugAttachDetector - Unable to start DebugAttachDetector, please add `--add-exports java.base/jdk.internal.vm=ALL-UNNAMED` to VM options 
2021-10-30 19:18:05,360 [  30272]   WARN - l.NotificationGroupManagerImpl - Notification group CodeWithMe is already registered (group=com.intellij.notification.NotificationGroup@5ba85b93). Plugin descriptor: PluginDescriptor(name=Code With Me, id=com.jetbrains.codeWithMe, descriptorPath=plugin.xml, path=~/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.clion/clion/2021.2/e059c730f99a229487162423f997293009985a5c/clion-2021.2/plugins/cwm-plugin, version=212.4746.93, package=null) 
2021-10-30 19:18:14,812 [  39724]   WARN - til.ui.update.LazyUiDisposable - use application as a parent disposable 
2021-10-30 19:18:28,977 [  53889]   WARN - .intellij.diagnostic.VMOptions - VM options file not configured 
2021-10-30 19:18:48,478 [  73390]   WARN - ystem.impl.ActionPopupMenuImpl - 1155 ms to fill popup menu EditorPopup 
2021-10-30 19:19:16,327 [ 101239]  ERROR - llij.ide.plugins.PluginManager - Could not initialize class kotlinx.coroutines.CoroutineExceptionHandlerImplKt 
java.lang.NoClassDefFoundError: Could not initialize class kotlinx.coroutines.CoroutineExceptionHandlerImplKt
	at kotlinx.coroutines.CoroutineExceptionHandlerKt.handleCoroutineException(CoroutineExceptionHandler.kt:33)
	at kotlinx.coroutines.DispatchedTask.handleFatalException(DispatchedTask.kt:146)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:115)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:886)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:755)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:442)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:441)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2021-10-30 19:19:16,328 [ 101240]  ERROR - llij.ide.plugins.PluginManager - CLion 2021.2  Build #CL-212.4746.93 
2021-10-30 19:19:16,328 [ 101240]  ERROR - llij.ide.plugins.PluginManager - JDK: 11.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2021-10-30 19:19:16,328 [ 101240]  ERROR - llij.ide.plugins.PluginManager - OS: Mac OS X 
2021-10-30 19:19:16,328 [ 101240]  ERROR - llij.ide.plugins.PluginManager - Plugin to blame: UTBotCppClion version: 0.0.1 
2021-10-30 19:19:16,329 [ 101241]  ERROR - llij.ide.plugins.PluginManager - Last Action: com.github.vol0n.utbotcppclion.actions.GenerateForFunctionAction 
```